src/KafkaFlow.Retry.UnitTests/KafkaFlow.Retry/Durable/RetryDurableConsumerLatestMiddlewareTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace KafkaFlow.Retry.UnitTests.KafkaFlow.Retry.Durable
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
+    using FluentAssertions;
+    using global::KafkaFlow.Retry.Durable;
+    using global::KafkaFlow.Retry.Durable.Encoders;
+    using global::KafkaFlow.Retry.Durable.Repository;
+    using Moq;
+    using Xunit;
+
+    [ExcludeFromCodeCoverage]
+    public class RetryDurableConsumerLatestMiddlewareTests
+    {
+        public static readonly IEnumerable<object[]> DataTest = new List<object[]>
+        {
+            new object[]
+            {
+                null,
+                Mock.Of<IRetryDurableQueueRepository>(),
+                Mock.Of<IUtf8Encoder>()
+            },
+            new object[]
+            {
+                Mock.Of<ILogHandler>(),
+                null,
+                Mock.Of<IUtf8Encoder>()
+            },
+            new object[]
+            {
+                Mock.Of<ILogHandler>(),
+                Mock.Of<IRetryDurableQueueRepository>(),
+                null
+            }
+        };
+
+        private readonly Mock<ILogHandler> logHandler = new Mock<ILogHandler>();
+        private readonly Mock<IMessageContext> messageContext = new Mock<IMessageContext>();
+        private readonly Mock<IRetryDurableQueueRepository> retryDurableQueueRepository = new Mock<IRetryDurableQueueRepository>();
+        private readonly Mock<IUtf8Encoder> utf8Encoder = new Mock<IUtf8Encoder>();
+
+        [Theory]
+        [MemberData(nameof(DataTest))]
+        public void RetryDurableConsumerLatestMiddleware_Ctor_Tests(
+            object logHandler,
```

Comment:
These three arguments should be the three concrete objects instead of using `object`.

Reply:
Done